### ISSUE TITLE
Ability to set font size

### ISSUE THREAD

**Issue by ghostdevv** [mzohaibqc/svelte-toasts#3]
I would love to be able to set the font size without having to edit the `st-toast-title` and `st-toast-description` classes, via a prop I imagine

**Comment by mzohaibqc** [comment]
@ghostdevv I don't think so. If we follow this approach, we have to provide props for everything like margin, background, color, and even for different sections, so it will not scale well. IMO, overriding respective class is better solution.

```html
<div class="parent">
  <ToastContainer {toasts} let:data={data}>
    <FlatToast {data} />
  </ToastContainer>
</div>

<style>
  .parent :global(.st-toast-description) {
    color: red;
   font-size: 1rem;
  }
</style>
```

**Comment by ghostdevv** [comment]
@mzohaibqc Alright that makes sense, maybe instead of using `rem`s for sizing, `em` or even `px` could be used? I had to rework the fontsizing of my entire app to make rems work

**Comment by mzohaibqc** [comment]
You don't need to rework your whole app. Just override the toast related
font size with any unit like rem, px or em. I don't understand why you need
to refactor your whole app just to change toast styles.

**Comment by ghostdevv** [comment]
@mzohaibqc Everything in the FlatToast at least uses `rem`s for margins, padding etc etc - and my app by default was setting `1rem` equal to `8px`, but the default for browsers is `16px`. Since you would've designed everything using `16px` it really messed up the toast, so I had to set it to `16px` and then halve all of my rem values to compensate

**Comment by mzohaibqc** [comment]
You don't need to do that. Just add a parent container and override
FlatToast styles using :global like this

.parent :global(.st-toast-description) {
    font-size: 14px; // or 1em;
}

**Comment by ghostdevv** [comment]
@mzohaibqc that doesn't fix the issue of everything else using rems
